### ISSUE TITLE
Error opening csv downloaded from the roster upload list 

### ISSUE THREAD

**Issue by isabelcshaw** [OpenSRP/opensrp-client-reveal#1346]
**Description:** 
For some of the csv files that have been uploaded via the clients page, we are now seeing that there is a periodic error where when you download them and try to open them, the file just shows a message that reads "Sorry. The file you are looking for does not exist."

This is not a consistent issue and it's the first time we've seen it. 

It is working for some: 
Eswatini_Child_Roaster_March1.csv 
SK_valid_sample_upload_ntd.csv (uploaded yesterday) 

Not others: 
Child roster test 3.23.csv
Eswatini_Child_Roaster_20210406_SB23.csv
Eswatini_Child_Roaster_20210406_SB2.csv

**Steps to reproduce:** 
Upload a csv via the clients page 
See that csv in the list of "uploaded client lists" 
Click download 
**_Open the downloaded csv_** 

(Or just download the csv files shared above) 

@moshthepitt pls let me know if there is any other information I can share

**Comment by isabelcshaw** [comment]
CHAI shared the csv files that are throwing the error (referenced above) and we were able to upload, download, and open the csv successfully. This has been closed because it can't be reproduced and the files seemed acceptable, but we still don't have any explanation or assurance it won't happen again.

**Comment by isabelcshaw** [comment]
Since this last comment, we have been able to reproduce the issue with csv files shared by CHAI. When I test this feature, I am able to successfully download and open any csv that I have made and uploaded. But, I have seen that when I upload and download a csv from CHAI, that the error is thrown upon opening it. I have also seen that error when I try to download and open the csv files shared above. 

To be clear, the upload is working fine because child clients are being generated. And the download feature below, where you download a csv based on the location hierarchy is also working fine. AND the download feature does work some of the time. But what CHAI has shared is definitely reoccurring. 

We have asked them to create csv files in Google Sheets and download them and upload them to Reveal without opening them to make sure there isn't an error due to their csv editor, and they have done so, but the problem persists.
